jooby/src/main/java/io/jooby/SslOptions.java

```diff
@@ -72,6 +87,8 @@
 
   private ClientAuth clientAuth = ClientAuth.NONE;
 
+  private List<String> protocol = Arrays.asList(TLS_V1_2, TLS_V1_3);
```

Comment:
It may be better to list 1.3 first, if a server implementation takes care about the order.

Reply:
Not sure if they care, but I'm reverting.